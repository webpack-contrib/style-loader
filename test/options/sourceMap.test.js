/* eslint-disable */
import path from 'path';
import webpack from '../helpers/compiler';

describe.skip('Options', () => {
  describe('sourceMap', () => {
    test('{Boolean}', async () => {
      const config = {
        rules: [
          {
            test: /\.css$/,
            use: [
              {
                loader: path.resolve(__dirname, '../../src'),
                options: {
                  sourceMap: true
                }
              },
              {
                loader: 'css-loader',
                options: {
                  sourceMap: true
                }
              }
            ]
          }
        ]
      };

      const stats = await webpack('fixture.js', config);
      let { source } = stats.toJson().modules[3];

      source = source.replace(process.cwd(), '');

      expect(source).toMatchSnapshot();
    });
  });
});
