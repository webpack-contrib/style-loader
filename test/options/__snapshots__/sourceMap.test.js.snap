// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Options sourceMap {Boolean} 1`] = `
"exports = module.exports = require(\\"../../node_modules/css-loader/lib/css-base.js\\")(true);
// imports


// module
exports.push([module.id, \\".class {\\\\n  color: red;\\\\n}\\\\n\\", \\"\\", {\\"version\\":3,\\"sources\\":[\\"/test/fixtures/fixture.css\\"],\\"names\\":[],\\"mappings\\":\\"AAAA;EACE,WAAW;CACZ\\",\\"file\\":\\"fixture.css\\",\\"sourcesContent\\":[\\".class {\\\\n  color: red;\\\\n}\\\\n\\"],\\"sourceRoot\\":\\"\\"}]);

// exports
"
`;
